CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+# CorisxTH Contributing Guidelines #
+
+## Tell other devs what you are working on! ##
+
+When you start working on a fix/enhancement for an open issue always post a
+comment in this issue's discussion to tell other devs that you have started
+working on it so that they won't waste any of their free time by working on
+their own fix/enhancement for it.
+
+If there isn't an open issue for a bug fix/enhancement you want to work on
+please check that one wasn't closed by a previous developer who didn't finish
+their work, which you could finish.
+
+And if there is no existing issue the bug fix or feature you plan to contribute
+open a new issue for it and tell other devs here that your going to/have
+started working on it.
+
+**If you have an idea for an enhancement that does not have a discussion, make
+a *Feature Request* first to discuss its desirability.  This helps make sure
+your development time is a worthy endeavour.**
+
+## Enhancement/Suggestions ##
+
+If you want to make a suggestion, please use the *Feature Request* option when
+making a new issue. It will provide all the information needed to evaluate the
```

Comment:
```suggestion
To suggest an enhancement, open a new [issue](https://github.com/CorsixTH/CorsixTH/issues/new/choose)
and click the *Feature Request* button.\
It will provide all the information needed to evaluate the
```